wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -581,6 +637,38 @@ private static function findStateField(array $array) {
     return $stateItemIndex !== FALSE ? $filteredItems[$stateItemIndex] : FALSE;
   }
 
+  /**
+   *
+   */
+  private static function findContactId(array $array) {
```

Comment:
Can we make this a bit more specific. A collection camp can be connected with many contacts like:

1. Initiators
2. Event Volunteer
3. Person who authorized it,
4. etc.

So it's better to use a more specific name like: `findCollectionCampInitiatorContact`